include/deal.II/grid/grid_tools.h

```diff
@@ -1024,6 +1046,31 @@ namespace GridTools
   vertex_to_cell_map(const Triangulation<dim,spacedim> &triangulation);
 
   /**
+   * Returns a normalized vector that contains a tensor for every vertex-cell
```

Comment:
the tensors are normalized right? Can you rephrase the text to say something like "a vector of normalized tensors for each vertex for each cell" if that is correct.